test/test_io.py

```diff
@@ -8,20 +8,28 @@
 
 def test_netcdf_io(scipy_network, tmpdir):
```

Comment:
maybe also test empty and layered dictionaries:

```suggestion
import pytest

@pytest.mark.parametrize("meta", [{"test": "test"}, {"test": {"test": "test"}}])
def test_netcdf_io(scipy_network, tmpdir, meta):
```

(analogous also for hdf5 and csv IO tests)